src/index.ts

```diff
@@ -118,6 +123,56 @@ export function main(argString = ""): void {
     );
   }
 
+  if (stashItems.length > 0) {
+    if (
+      userConfirmDialog(
+        `Garbo has detected that you have the following items still out of the stash from a previous run of garbo: ${stashItems
+          .map((item) => item.name)
+          .join(", ")}. Would you like us to return these to the stash now?`,
+        true
+      )
+    ) {
+      startSession();
+      try {
+        const clanIdOrName = globalOptions.prefs.stashClan;
+        const parsedClanIdOrName =
+          clanIdOrName !== "none"
+            ? clanIdOrName.match(/^\d+$/)
+              ? parseInt(clanIdOrName)
+              : clanIdOrName
+            : null;
+
+        if (parsedClanIdOrName) {
+          Clan.with(parsedClanIdOrName, () => {
+            for (const item of [...stashItems]) {
+              if (getFoldGroup(item).some((item) => have(item))) {
+                cliExecute(`fold ${item}`);
+              }
+              const retrieved = retrieveItem(item);
+              if (
+                item === $item`Spooky Putty sheet` &&
+                !retrieved &&
+                have($item`Spooky Putty monster`)
+              ) {
+                continue;
+              }
+              print(`Returning ${item} to ${getClanName()} stash.`, HIGHLIGHT);
+              if (putStash(item, 1)) {
+                stashItems.splice(stashItems.indexOf(item), 1);
+              }
+            }
+          });
+        } else throw new Error("Error: No garbo_stashClan set.");
+      } finally {
+        endSession(false);
+      }
+    } else {
+      stashItems.splice(0, stashItems.length);
```

Comment:
I think there's a degenerate case here.

Let's say that you end the day without returning items to the stash.
The next day, you run the script and are prompted to return the items.
You select no.
The `stashItems` array is then emptied.
At the end of this session, the `garboStashItems` property is updated to be the `stashItems` array, which should empty it as well?
If you want to then return items to the clan stash, you have to do so manually, don't you?